draft-ietf-quic-multipath.md

```diff
@@ -1422,7 +1422,8 @@ in {{QUIC-TRANSPORT}}.
 
 This document does not specify per-path idle timeouts. An endpoint
 can decide to close a path at any time, whether the path is in active
-use or not, by sending a PATH_ABANDON frame. It is not required
+use or not, by sending a PATH_ABANDON frame. For the initial PATH_ABANDON
+frame to close a path, it is not required
 to send a PATH_ABANDON frame at any specific point in time.
```

Comment:
The sentence "For the initial PATH_ABANDON
frame to close a path, it is not required
to send a PATH_ABANDON frame at any specific point in time" is rather hard to parse. How about removing it entirely? We have already said that "An endpoint
can decide to close a path at any time."

Reply:
I marking this comment as resolved now because you it added in somewhere in the middle and that stops me from suggesting a change. Please see the change I will suggest next and let me know that's better. Otherwise we can maybe also remove this part.